test/commands/scanner/rule/describe.test.ts

```diff
@@ -20,8 +20,8 @@ describe('scanner:rule:describe', () => {
 				.it('--json flag yields correct results', ctx => {
 					const ctxJson = JSON.parse(ctx.stdout);
 					expect(ctxJson.result.length).to.equal(0, 'Should be no results');
-					expect(ctxJson.warnings.length).to.equal(1, 'Should be one warning');
-					expect(ctxJson.warnings[0]).to.equal(formattedWarning, 'Warning message should match');
+					expect(ctxJson.warnings.length).to.equal(2, 'Should be one warning');
```

Comment:
The failure text says "one" instead of "two".

Reply:
Will fix this